main/webapp/modules/core/scripts/dialogs/clustering-dialog.js

```diff
@@ -175,7 +175,9 @@ ClusteringDialog.prototype._renderTable = function(clusters) {
               cluster.value = value;
               
               parent.find("input[type='text']").val(value);
-              parent.find("input[type='checkbox']").attr('checked', true).change();
+              var checkbox = parent.find("input[type='checkbox']");
+              if (!checkbox.attr('checked'))
```

Comment:
Conditionalizing this doesn't fix the underlying issue which is that the internal state doesn't track the state of the checkbox.  Instead the fix should be applied at line 222.